apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -140,10 +141,7 @@ const SummaryResponses = ({
     studentWork: response.text,
   }));
 
-  const AiEvaluationMVPUnits = ['csp4-2024', 'csp6-2024', 'allthethings'];
-  const aiAnalysisAvailable = AiEvaluationMVPUnits.includes(
-    scriptData.reportingData.unitName
-  );
+  const aiAnalysisAvailable = hasAiAnalysis;
```

Comment:
we can pick one variable name `hasAiAnalysis` or `aiAnalysisAvailable` set it to `scriptData.show_ai_analysis` and then use it everywhere in this file

Reply:
Good point!  Changed!